<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">zxtune-qt</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/zxtune.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>1</number>
    </property>
    <property name="spacing">
     <number>1</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>24</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuLayout">
    <property name="title">
     <string>Layout</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAddFiles"/>
    <addaction name="actionAddFolder"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionComponents"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
    <addaction name="separator"/>
    <addaction name="actionCheckUpdates"/>
    <addaction name="separator"/>
    <addaction name="actionOnlineHelp"/>
    <addaction name="actionWebSite"/>
    <addaction name="actionOnlineFAQ"/>
    <addaction name="actionReportBug"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLayout"/>
  </widget>
  <action name="actionAddFiles">
   <property name="icon">
    <iconset>
     <normalon>:/playlist/addfiles.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Add files</string>
   </property>
   <property name="shortcut">
    <enum>QKeySequence::Open</enum>
   </property>
  </action>
  <action name="actionAddFolder">
   <property name="icon">
    <iconset>
     <normalon>:/playlist/addfolder.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Add folder</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Alt+O</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset>
     <normalon>:/application/preferences.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <enum>QKeySequence::Preferences</enum>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normalon>:/application/exit.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionOnlineHelp">
   <property name="icon">
    <iconset>
     <normalon>:/application/help.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Online help</string>
   </property>
   <property name="shortcut">
    <enum>QKeySequence::HelpContents</enum>
   </property>
  </action>
  <action name="actionWebSite">
   <property name="icon">
    <iconset>
     <normalon>:/application/website.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Visit web site</string>
   </property>
  </action>
  <action name="actionOnlineFAQ">
   <property name="icon">
    <iconset>
     <normalon>:/application/webfaq.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Online FAQ</string>
   </property>
  </action>
  <action name="actionReportBug">
   <property name="icon">
    <iconset>
     <normalon>:/application/reportbug.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Report bug online</string>
   </property>
  </action>
  <action name="actionCheckUpdates">
   <property name="icon">
    <iconset>
     <normalon>:/application/checkupdates.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Check updates</string>
   </property>
  </action>
  <action name="actionComponents">
   <property name="icon">
    <iconset>
     <normalon>:/application/components.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Components</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normalon>:/application/about.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset>
     <normalon>:/application/aboutqt.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>About QT</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
  </connection>
 </connections>
</ui>
